__tests__/app.test.js

```diff
@@ -48,12 +48,15 @@ describe("GET", () => {
             .get('/api/articles/4')
             .expect(200)
             .then(({body}) => {
-                expect(body.article).toEqual({article_id: 4, 
-                    article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700", 
-                    author: "rogersop", 
-                    body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages", created_at: "2020-05-06T01:14:00.000Z", 
-                    title: "Student SUES Mitch!", topic: "mitch", 
-                    votes: 0})
+                expect(body.article).toHaveProperty('author', expect.any(String))
+                expect(body.article).toHaveProperty('title', expect.any(String))
+                expect(body.article).toHaveProperty('article_id', expect.any(Number))
+                expect(body.article).toHaveProperty('body', expect.any(String))
+                expect(body.article).toHaveProperty('topic', expect.any(String))
+                expect(body.article).toHaveProperty('created_at', expect.any(String))
+                expect(body.article).toHaveProperty('votes', expect.any(Number))
+                expect(body.article).toHaveProperty('article_img_url', expect.any(String))
+                expect(body.article).toHaveProperty('comment_count', expect.any(Number));
```

Comment:
Nearly there! If you remove `comment_count` from this test, then have a new test showing that comment_count has been added to the endpoint, that'll be fine 👍